test/integration/model/schema.test.js

```diff
@@ -0,0 +1,458 @@
+          return current.createSchema('schema_two');
+        }).then(function() {
+          return self.RestaurantOne.sync({force: true});
+        }).then(function() {
+          return self.RestaurantTwo.sync({force: true});
+        }).catch(function(err) {
+          expect(err).to.be.null;
+        });
+      });
+
+      afterEach('drop schemas', function() {
+        return current.dropSchema('schema_one').then(function() {
+          return current.dropSchema('schema_two');
+        });
+      });
+
+      describe('Add data via model.create, retrieve via model.findOne', function() {
+        it('should be able to insert data into the table in schema_one using create', function() {
+          var self = this;
+          var restaurantId;
+
+          return self.RestaurantOne.create({
+            foo: 'one',
+            location_id: locationId
+          }).then(function() {
+              return self.RestaurantOne.findOne({
+                where: {foo: 'one'}});
+            }).then(function(obj) {
+              expect(obj).to.not.be.null;
+              expect(obj.foo).to.equal('one');
+              restaurantId = obj.id;
+              return self.RestaurantOne.findById(restaurantId);
+            }).then(function(obj) {
+              expect(obj).to.not.be.null;
+              expect(obj.foo).to.equal('one');
+            });
+        });
+
+        it('should fail to insert data into schema_two using create', function() {
+          var self = this;
+
+          return self.RestaurantTwo.create({
+            foo: 'test'
+          }).catch(function(err) {
+            expect(err).to.not.be.null;
+          });
+        });
+
+        it('should be able to insert data into the table in schema_two using create', function() {
+          var self = this;
+          var restaurantId;
+
+          return self.RestaurantTwo.create({
+            foo: 'two',
+            location_id: locationId
+          }).then(function() {
+              return self.RestaurantTwo.findOne({
+                where: {foo: 'two'}});
+            }).then(function(obj) {
+              expect(obj).to.not.be.null;
+              expect(obj.foo).to.equal('two');
+              restaurantId = obj.id;
+              return self.RestaurantTwo.findById(restaurantId);
+            }).then(function(obj) {
+              expect(obj).to.not.be.null;
+              expect(obj.foo).to.equal('two');
+            });
+        });
+
+
+        it('should fail to find schema_one object in schema_two', function() {
+          var self = this;
+
+          return self.RestaurantTwo.findOne({where: {foo: 'one'}}).then(function(RestaurantObj) {
+            expect(RestaurantObj).to.be.null;
+          });
+        });
+
+        it('should fail to find schema_two object in schema_one', function() {
+          var self = this;
+
+          return self.RestaurantOne.findOne({where: {foo: 'two'}}).then(function(RestaurantObj) {
```

Comment:
These two tests should be moved to the 2 tests above them to verify I cannot get the data in the opposite schema.

- [ ] Combine opposite schema verification into create data tests